src/components/ScheduleRoom/ScheduleNavigation.vue

```diff
@@ -13,56 +12,89 @@ defineProps({
+  { label: t("label.months.november"), value: 11 },
+  { label: t("label.months.dezember"), value: 12 },
+];
+
+function selectMonth(month: number) {
+  emits("select-month", month);
+}
+
 const activedModal = ref(false);
+
+function reload() {
+  activedModal.value = false;
+  emits("reload");
+}
 </script>
 
 <template>
   <AddSchedule
     :is-active="activedModal"
     :rooms="rooms"
     @cancel="activedModal = false"
-    @reload="$emit('reload', (activedModal = false))"
+    @reload="reload"
   />
 
-  <div class="row justify-between full-width">
-    <div class="row q-pa-md text-black col-2">
-      <q-btn no-caps flat class="row bg-primary text-white button-border fit">
-        <q-btn-group flat class="column">
-          <q-btn
-            class="no-padding"
-            @click="$emit('prev')"
-            dense
-            icon="arrow_drop_up"
-          />
-          <q-btn
-            class="no-padding"
-            @click="$emit('next')"
-            dense
-            icon="arrow_drop_down"
-          />
-        </q-btn-group>
-        <span class="text-h5">
-          {{
-            DateTime.fromFormat(date as string, "yyyy-MM-dd").setLocale("pt-br")
-              .monthLong
-          }}
-        </span>
-      </q-btn>
+  <div class="row justify-between q-mx-lg">
+    <div class="column q-pa-md">
+      <q-btn-group class="row justify-center q-px-xs">
+        <NavigationButton icon="arrow_left" @click="$emit('prev')" />
+        <q-btn-dropdown
+          transition-show="scale"
+          transition-hide="scale"
+          no-caps
+          text-color="primary"
+          class="text-h6 q-ml-lg"
+          dropdown-icon="false"
+          :label="month"
+        >
+          <q-list v-for="(month, index) in months" :key="index">
+            <q-item
+              clickable
+              v-close-popup
+              @click="selectMonth(month.value)"
+              class="text-black text-h6"
+              v-model="month.value"
+            >
+              <q-item-section>
+                <q-item-label>{{ month.label }}</q-item-label>
+              </q-item-section>
+            </q-item>
+          </q-list>
+        </q-btn-dropdown>
+        <NavigationButton icon="arrow_right" @click="$emit('next')" />
+      </q-btn-group>
     </div>
-
-    <div class="row q-py-md text-black col-4">
-      <q-btn-group class="border-rounded col-12">
-        <q-btn no-caps class="col-4" :label="t('label.date.month')" />
-        <q-btn no-caps class="col-4" :label="t('label.date.week')" />
-        <q-btn no-caps class="col-4" :label="t('label.date.day')" />
+    <!-- TODO esses botões vão ganhar funcionalidades futuramente -->
+    <div class="row q-py-md col-4">
+      <q-btn-group class="rounded col-10">
+        <NavigationButton class="col-4" :label="t('label.date.month')" />
+        <NavigationButton class="col-4" :label="t('label.date.week')" />
+        <NavigationButton class="col-4" :label="t('label.date.day')" />
```

Comment:
Os botoes não possuem funcionalidade? Se a resposta for negativa, por que eles estão sendo exibidos sem nenhuma funcionalidade?